src/components/LogViewer/LogViewer.tsx

```diff
@@ -162,7 +162,7 @@ export const LogViewer = ({ isStatic = false, data, customActions }: ILogViewerP
       data={renderedData}
       onScroll={onScroll}
       toolbar={<HeaderToolbar />}
-      footer={isPaused && !isFollowing && <FooterButton />}
+      footer={!isStatic && isPaused && !isFollowing && <FooterButton />}
```

Comment:
when scrolling up for a static log(like finished deliverable analysis), there is still a resume button, which I think should be removed.